### PR TITLE
AArch64: Implement adjustReturnValue in JNILinkage

### PR DESCRIPTION
**Pull request by Akira1Saitoh**
Implement `adjustReturnValue` helper method in JNILinkage for aarch64.

Depends on https://github.com/eclipse/openj9/pull/7972

Signed-off-by: Akira Saitoh <saiaki@jp.ibm.com>

### CODE REVIEW

**Comment by Akira1Saitoh** [comment]
Un-wipping as we talked privately today, so that this PR can be reviewed once https://github.com/eclipse/openj9/pull/7972 is merged.

**Comment by Akira1Saitoh** [comment]
Rebased on master.

File: runtime/compiler/aarch64/codegen/ARM64JNILinkage.cpp [hunk 1]
```diff
@@ -111,7 +111,57 @@ TR::Register *J9::ARM64::JNILinkage::pushJNIReferenceArg(TR::Node *child)
...
+   // jni methods may not return a full register in some cases so need to get the declared
+   // type so that we sign and zero extend the narrower integer return types properly
+   TR::LabelSymbol *tempLabel = generateLabelSymbol(cg());
+
+   switch (resolvedMethod->returnType())
+      {
+      case TR::Address:
+         if (wrapRefs)
+            {
+            // unwrap the return object is non-null
```
Comment (knn-k) [inline runtime/compiler/aarch64/codegen/ARM64JNILinkage.cpp]: Perhaps you mean "unwrap **when** the return object is non-null" ?

Comment (Akira1Saitoh) [reply runtime/compiler/aarch64/codegen/ARM64JNILinkage.cpp]: Yes. I corrected the comment.

**Comment by knn-k** [comment]
jenkins compile alinux64xl jdk11

**Pull request merged**